core/app/models/spree/refund.rb

```diff
@@ -29,26 +26,32 @@ def total_amount_reimbursed_for(reimbursement)
       end
     end
 
+    def perform_and_save!
```

Comment:
Can we add some RDoc to this method, letting people know that they should always use this to save a refund unless they are importing data? (Until we do something like add state column and perhaps can get rid of this method entirely)